sorc/ncep_post.fd/PARA_RANGE.f

```diff
@@ -46,4 +46,42 @@ SUBROUTINE PARA_RANGE (N1,N2,NPROCS,IRANK,ISTA,IEND)
       if ( iwork2 > irank ) iend = iend + 1
       return
       end
+!!
+!! USAGE:    CALL PARA_RANGE2(N1,N2,NPROCS,IRANK,ISTA,IEND)(A)
```

Comment:
There are 9 input arguments for para_range2,  I suggest correct this line to:

!! USAGE:    CALL PARA_RANGE2(N1,N2,NX,NY,NRANK,ISTA,IEND)

This would be consistent with the below description.